src/Compilers/CSharp/Portable/Emitter/Model/MostCommonNullableValueBuilder.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal struct MostCommonNullableValueBuilder
+    {
+        private int _0;
+        private int _1;
+        private int _2;
+
+        internal byte? MostCommonValue
+        {
+            get
+            {
+                int max;
+                byte b;
+                if (_1 > _0)
```

Comment:
>_0 [](start = 25, length = 2)

Names like this are confusing. It looked to me as just '''0''' literal here and confused me.